mne/viz/topo.py

```diff
@@ -819,8 +823,9 @@ def plot_topo_image_epochs(epochs, layout=None, sigma=0., vmin=None,
     vmax : float
         The max value in the image. The unit is uV for EEG channels,
         fT for magnetometers and fT/cm for gradiometers.
-    colorbar : bool
-        Display or not a colorbar.
+    colorbar : bool | None
+        Display or not a colorbar. If ``None`` (the default) a colorbar will be
+        shown only if all channels are of the same type.
```

Comment:
When chenging the default you could do a pass so that all of them follow the recomended starnad. That shows the default at the end. here are some examples:

```diff
-     colorbar : bool | None
-        Display or not a colorbar. If ``None`` (the default) a colorbar will be
-        shown only if all channels are of the same type.
+    colorbar : bool | None
+        Display or not a colorbar. If ``None`` a colorbar will be
+        shown only if all channels are of the same type. Defaults to ``None``
```

```diff
-    show : bool
-        Show figure if True.
+    show : Bool
+        Weather to show the figure or not. Defaults to ``True``.
```